vochain/account.go

```diff
@@ -392,6 +399,98 @@ func (v *State) setDelegate(accountAddr, delegateAddr common.Address, txType mod
 	}
 }
 
+func SendTokensTxCheck(vtx *models.Tx, txBytes, signature []byte, state *State) (common.Address, common.Address, uint64, uint32, error) {
```

Comment:
I know returning many individual parameters is a quite common behavior here. However I think we should slowly change it. So IMO it would be better if this method returns a struct+error instead of 5 unamed parameters.